File: CustomAdaptorExam/app/src/main/java/com/example/kyungjoon/customadaptorexam/MainActivity.java
package com.example.kyungjoon.customadaptorexam;

import android.app.ListActivity;
import android.app.Notification;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import java.util.ArrayList;

public class MainActivity extends ListActivity {

    // declare class variables
    private ArrayList<Item> m_parts = new ArrayList<Item>();
    private Runnable viewParts;
    private ItemAdapter m_adapter;

    /** Called when the activity is first created. */
    public void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);

        // instantiate our ItemAdapter class
        m_adapter = new ItemAdapter(this, R.layout.list_item, m_parts);
        setListAdapter(m_adapter);

        // here we are defining our runnable thread.
        viewParts = new Runnable(){
            public void run(){
                handler.sendEmptyMessage(0);
            }
        };

        // here we call the thread we just defined - it is sent to the handler below.
        Thread thread =  new Thread(null, viewParts, "MagentoBackground");
        thread.start();
    }

    private Handler handler = new Handler()
    {
        public void handleMessage(Notification.MessagingStyle.Message msg)
        {
            // create some objects
            // here is where you could also request data from a server
            // and then create objects from that data.
            m_parts.add(new Item("MyItemName", "This is item #1", 0));
            m_parts.add(new Item("MyItemName #2", "This is item #2", 0));

            m_adapter = new ItemAdapter(MainActivity.this, R.layout.list_item, m_parts);

            // display the list.
            setListAdapter(m_adapter);
        }
    };
}

File: app/src/main/java/com/findmyage/kyungjoon/findmyage/MainActivity.kt
package com.findmyage.kyungjoon.findmyage

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

       /* buFindAge.setOnClickListener{



        }*/
    }

    fun buFindAgeEvent(view: View){
        //버튼이 클릭 되었을떄요..

        val yearOfBitrh:Int = textYearOfBirth.text.toString().toInt();

        var myAge = 2017-yearOfBitrh;

        txtShowAge.text = "내나이는 년도는  "+ myAge;
    }
}
